<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallDetails</class>
 <widget class="QDialog" name="CallDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Call Details</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="5">
    <widget class="QFrame" name="frBBControls">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPushButton" name="btCallDetailsCloseBB">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="btCallDetailsPlayBB">
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="bbFrameLabel">
          <property name="text">
           <string>Call details</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="btCallDetailsStopBB">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QFrame" name="frDesktopControls">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QPushButton" name="btCallDetailsStop">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btCallDetailsClose">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btCallDetailsPlay">
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QFrame" name="frCallDetailsTools">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="btCallDetailsDelete">
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset resource="taki.qrc">
          <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="btCallDetailsAddContact">
        <property name="text">
         <string>Add contact</string>
        </property>
        <property name="icon">
         <iconset resource="taki.qrc">
          <normaloff>:/icons/add-contact.png</normaloff>:/icons/add-contact.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="btCallDetailsBack">
        <property name="text">
         <string>Back</string>
        </property>
        <property name="icon">
         <iconset resource="taki.qrc">
          <normaloff>:/icons/arrow-left.png</normaloff>:/icons/arrow-left.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <layout class="QGridLayout" name="gridLayout_6">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QTextBrowser" name="callDetailsText">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
       <property name="openLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="taki.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btCallDetailsClose</sender>
   <signal>clicked()</signal>
   <receiver>CallDetails</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCallDetailsStop</sender>
   <signal>clicked()</signal>
   <receiver>CallDetails</receiver>
   <slot>stopPlayback()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCallDetailsPlay</sender>
   <signal>clicked()</signal>
   <receiver>CallDetails</receiver>
   <slot>startPlayback()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCallDetailsCloseBB</sender>
   <signal>clicked()</signal>
   <receiver>CallDetails</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCallDetailsPlayBB</sender>
   <signal>clicked()</signal>
   <receiver>CallDetails</receiver>
   <slot>startPlayback()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCallDetailsStopBB</sender>
   <signal>clicked()</signal>
   <receiver>CallDetails</receiver>
   <slot>stopPlayback()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCallDetailsBack</sender>
   <signal>clicked()</signal>
   <receiver>CallDetails</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>13</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>stopPlayback()</slot>
  <slot>startPlayback()</slot>
  <slot>addCallDetailToContact()</slot>
  <slot>delCallDetail()</slot>
 </slots>
</ui>
